db/migrate/20161203210855_create_venues.rb

```diff
@@ -0,0 +1,13 @@
+class CreateVenues < ActiveRecord::Migration[5.0]
+  def change
+    create_table :venues do |t|
+      t.string :name
+      t.string :address
+      t.string :maps_url
```

Comment:
sorry.. i made typo in issue description `maps_url`. Could you please change to `map_url` ?